shell/app/modules/project/pages/test-plan/test-plan.tsx

```diff
@@ -161,6 +176,20 @@ const TestPlan = () => {
 
   const filterConfig: FilterItemConfig[] = React.useMemo(
     () => [
+      {
+        type: Select,
+        name: 'isArchived',
+        customProps: {
+          options: archiveStatusMap.map(({ label, value }) => (
+            <Option key={value} value={value}>
+              {label}
+            </Option>
+          )),
+          allowClear: true,
+          placeholder: i18n.t('project:archive status'),
+          mode: 'multiple',
```

Comment:
`multiple` same to be confused here, what's the difference between select all and select none.